### ISSUE TITLE
Behavior of recording settings box in the bottom right corner of the screen

### ISSUE THREAD

**Issue by ceskyDJ** [alyssaxuu/screenity#200]
Hi @alyssaxuu! Your animations and very detail and responsive behavior of recording settings box are just incredible! The is just one little imperfection — when I move the box to the bottom right corner of my screen and click on "Show more options" (in the bottom of the box), the context menu isn't visible (it's "out of screen"). Could you add some automatic movement like when I try to more the box "outside the window" to handle this situation, please?

There are some additional screenshots, I hope they will help to understand what I mean:
![image](https://github.com/alyssaxuu/screenity/assets/48548230/c09c4d1e-8457-45a7-a490-9bdc45f86409)
![image](https://github.com/alyssaxuu/screenity/assets/48548230/9e0e7673-f945-4185-9b87-60a4cf303a29)

**Comment by ceskyDJ** [comment]
There is one more thing. When i grab expanded settings box and try to put "outside the screen", it ends like this (with the "Help" partly outside of the screen):
![image](https://github.com/alyssaxuu/screenity/assets/48548230/912d4d5d-99e6-4c07-83dd-71eab5d0376e)

**Comment by alyssaxuu** [comment]
Yeah, I can see how this might be annoying, I'll look into moving it out of the way when it gets in this state.
